wazuh_filebeat/metadata.rb

```diff
@@ -6,7 +6,6 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.1.1'
 
-
```

Comment:
`wazuh_filebeat` is missing these lines.  Was this intentional or should these be included here?
```
issues_url 'https://github.com/wazuh/wazuh-chef/issues'
source_url 'https://github.com/wazuh/wazuh-chef'
chef_version '>= 12.14'
```